database/csvSeed.js

```diff
@@ -0,0 +1,102 @@
+const fs = require('fs');
+const csvWriter = require('csv-write-stream');
+const { Review, Item, sequelize} = require('./model.js');
+
+const { getFakeItem } = require('./utils/itemsSeed.js');
+const { getFakeReview } = require('./utils/reviewsSeed.js');
+
+
+// creating the items and review tables (empty)
+sequelize.sync({ force: true })
+  .then(() => {
+    console.log('mysql tables were created');
+  })
+  .catch((err) => {
+    console.log('could not create');
+  })
+  .finally(() => {
+    sequelize.close();
+  });
+
+
+const itemsToGenerate = 10000000;
```

Comment:
100% not needed, but it could help readability by implementing the new "Numeric Separators" functionality. E.g. 10_000_000 instead of 10000000.